### ISSUE TITLE
Filter instances using label selector instead of name prefix

### ISSUE THREAD

**Issue by jooola** [testflows/TestFlows-GitHub-Hetzner-Runners#39]
I recommend you using labels and label selector to group your instances, instead of relying on the name of the instances.

This way, filtering can happen server side, and you can have multiple instances groups in the same project.

**Comment by vzakaznikov** [comment]
Yes, we need to move to using labels and label selectors instead of pulling all the servers and checking their name prefix.
